### PR TITLE
mirror issues to ADAP jira project

### PR DESCRIPTION
**Pull request by nathaniel-may**
I'm attempting to mirror all the adapter issues into the new ADAP jira project. I found these new more generalized actions and I think this is how you would call them.

If this gets approved I'll open identical PRs in the rest of the adapter repos and wait to merge all the automation changes all at once.

### CODE REVIEW

File: .github/workflows/jira-transition.yml [hunk 1]
```diff
@@ -17,7 +17,9 @@ on:
```
[the remark this answers is not in the document]
Comment (emmyoop) [reply .github/workflows/jira-transition.yml:17]: I'm not sure how the adapters are configured.  I believe @leahwicz configured `dbt-core` to have no permission so we have to explicitly define permissions (even read) in all workflows.  If the adapter repos are not also set up this way, they should be.  Workflows inherit permissions so even though the workflow you call has a read permission set, if this one doesn't it won't be able to read.

tldr - you should define the permission in each workflow

**Review by emmyoop — CHANGES_REQUESTED** [review]
Permissions needs to be set but otherwise looks good.

I've added an [issue](https://github.com/dbt-labs/dbt-core/issues/6955) in core to update the workflows there and clean up the extra repository.

File: .github/workflows/jira-transition.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (nathaniel-may) [reply .github/workflows/jira-transition.yml:17]: makes sense. I copied the permission that core uses for the equivalent action.

**Review by nssalian — APPROVED** [review]
